python/paddle/tensor/manipulation.py

```diff
@@ -3203,6 +3300,123 @@ def unique_consecutive(
         return tuple(outs)
 
 
+@overload
+def unique(
+    x: Tensor,
+    return_index: Literal[True] = ...,
+    return_inverse: Literal[True] = ...,
+    return_counts: Literal[True] = ...,
+    axis: int | None = None,
```

Comment:
```suggestion
    axis: int | None = ...,
```

所有 overload 里删掉默认值吧，我们只在实现里写默认值